src/vs/workbench/api/browser/mainThreadComments.ts

```diff
@@ -185,13 +185,16 @@ export class MainThreadCommentThread<T> implements languages.CommentThread<T> {
 		public threadId: string,
 		public resource: string,
 		private _range: T | undefined,
+		comments: languages.Comment[] | undefined,
```

Comment:
Should that have a visibility ?

Reply:
There's actually a property in this class already called `comments`, so this is just an argument with no visibility needed.